sys-block/open-iscsi/files/iscsidev.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# KERNEL=="sd*", SUBSYSTEMS=="block", RUN{program}="/etc/udev/scripts/iscsidev.sh"
+
+# we only care about iscsi devices
+[[ $ID_VENDOR = "IET" ]] || exit 1
+
+# don't care about partitions either
+echo $DEVNAME | egrep -q "[0-9]$"
+status=$?
+[[ $status != 0 ]] || exit 1
+
+#ID_MODEL=VIRTUAL-DISK
+#ID_MODEL_ENC=VIRTUAL-DISK
+#ID_REVISION=0001
+#DEVTYPE=disk
+#ID_BUS=scsi
+#SUBSYSTEM=block
+#ID_SERIAL=1IET_00010001
+#DEVPATH=/devices/platform/host74/session68/target74:0:0/74:0:0:1/block/sde
+#ID_VENDOR_ENC=IET\x20\x20\x20\x20\x20
+#MINOR=64
+#ID_SCSI=1
+#ACTION=add
+#PWD=/
+#ID_PART_TABLE_UUID=54f71c65-a5d5-45cd-8915-5ffd5ff4fea6
+#ID_FS_TYPE=
+#USEC_INITIALIZED=999037905
+#MAJOR=8
+#ID_SCSI_SERIAL=beaf11
+#DEVLINKS=/dev/disk/by-id/scsi-1IET_00010001
+#DEVNAME=/dev/sde
+#SHLVL=1
+#ID_TYPE=disk
+#ID_PART_TABLE_TYPE=gpt
+#ID_VENDOR=IET
+#ID_SERIAL_SHORT=IET_00010001
+#SEQNUM=25775
+
+# do the removal
+if [[ $ACTION = 'remove' ]]; then
+  # nohup needed so this isn't constantly run...
+  nohup find -L /dev/disk/by-path/ -type l -lname ${DEVNAME} -exec rm "{}" + 2>/dev/null &
```

Comment:
Oh god, I hate this script and everything it stands for. This should really be implemented something that *just* sets gives udev env vars, run as `IMPORT{program}` and then leave managing symlinks to udev.